src/lz4.h

```diff
@@ -100,7 +100,7 @@ extern "C" {
 /*------   Version   ------*/
 #define LZ4_VERSION_MAJOR    1    /* for breaking interface changes  */
 #define LZ4_VERSION_MINOR    9    /* for new (non-breaking) interface capabilities */
-#define LZ4_VERSION_RELEASE  3    /* for tweaks, bug-fixes, or development */
+#define LZ4_VERSION_RELEASE  4    /* for tweaks, bug-fixes, or development */
```

Comment:
upgrading lz4 is trickier, as we have actually copied the entire code into our source tree. An update will look similar to https://github.com/confluentinc/librdkafka/pull/3148/files

Reply:
I was under the impression that most of the update has already happened, it was just the version update missing:
https://github.com/confluentinc/librdkafka/pull/4232

Reply:
oh, it was just the CVE fix, 
maybe it would be worthwhile to update lz4  as there are some significant performance gains for ARM64 platform:
https://github.com/lz4/lz4/releases